README.md

```diff
@@ -364,6 +364,22 @@ It **MAY** be a requirement and the server can send back `428 Precondition Requi
 This **MAY** be used for GET requests e.g don’t download data that hasn’t changed since the client last requested it
 
 
+###Remove properties of a resource
+Removes properties of a single resource using the PATCH verb and a subset (remove operation) of [JSON PATCH](https://tools.ietf.org/html/rfc6902) semantics.
+
+Content-Type must be set to application/json-patch+json
+
+*Example request*
+
+    PATCH /datasets/drafts/{id}
+    [
+     { "op": "remove", "path": "/name" }
```

Comment:
Well we can support others, but I think the intention was to stick to
`remove` only for now to make life easier for our clients.   We can discuss
extending this at the next steering group meeting.

On Tue, Apr 12, 2016 at 1:10 PM, Chris Howell <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Mendeley/api-design-documentation/pull/2#discussion_r59364268>
> :
>
> > @@ -364,6 +364,22 @@ It **MAY** be a requirement and the server can send back `428 Precondition Requi
> >  This **MAY** be used for GET requests e.g don’t download data that hasn’t changed since the client last requested it
> >
> >
> > +###Remove properties of a resource
> > +Removes properties of a single resource using the PATCH verb and a subset (remove operation) of [JSON PATCH](https://tools.ietf.org/html/rfc6902) semantics.
> > +
> > +Content-Type must be set to application/json-patch+json
> > +
> > +*Example request*
> > +
> > +    PATCH /datasets/drafts/{id}
> > +    [
> > +     { "op": "remove", "path": "/name" }
>
> is this still going to be the only operation supported.. now that you've
> got a library to do it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Mendeley/api-design-documentation/pull/2/files/d9f01eddf5c0aceba35ec3c92eb90375d228348b#r59364268>
>